Flow.Launcher/SettingWindow.xaml.cs

```diff
@@ -269,6 +269,23 @@ private void OpenLogFolder(object sender, RoutedEventArgs e)
         {
             PluginManager.API.OpenDirectory(Path.Combine(DataLocation.DataDirectory(), Constant.Logs, Constant.Version));
         }
+        private void ClearLogFolder(object sender, RoutedEventArgs e)
+        {
+            var confirmResult = MessageBox.Show(InternationalizationManager.Instance.GetTranslation("clearlogfolderMessage"),InternationalizationManager.Instance.GetTranslation("clearlogfolder"), MessageBoxButton.YesNo);
+            if (confirmResult == MessageBoxResult.Yes)
+            {
+                DirectoryInfo Di = new DirectoryInfo(Path.Combine(DataLocation.DataDirectory(), Constant.Logs, Constant.Version));
+                FileInfo[] files = Di.GetFiles();
```

Comment:
Maybe use EnumerateFiles?

Reply:
I don't know what it is. anyway, applied. 😅

Reply:
Ah it's just a newer api that has better performance (not noticable Though) by using state machine to allow process the result lazily. https://stackoverflow.com/questions/5669617/what-is-the-difference-between-directory-enumeratefiles-vs-directory-getfiles